### PR TITLE
PWX-33958: T34542903: Try with different drive sizes/types

### PR DESCRIPTION
**Pull request by ragsingh-px**
**What this PR does / why we need it**:
Automation test to create new pool with different types for vmware

**Which issue(s) this PR fixes** (optional)
Closes # PWX-33958

**Special notes for your reviewer**:
Jenkins test link:- https://jenkins.pwx.dev.purestorage.com/user/ragsingh@purestorage.com/my-views/view/all/job/Users/job/Raghvendra/job/raghav_pxdv-7-dev/108

### CODE REVIEW

**Review by vinayakshnd — APPROVED** [review]
LGTM

File: tests/basic/pool_function_test.go [hunk 1]
```diff
@@ -1423,3 +1421,62 @@ var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
...
+			log.InfoD("Validate pool rebalance after drive add to the node %s", storageNode.Name)
+			err = ValidateDriveRebalance(*storageNode)
+			log.FailOnError(err, "pool re-balance failed on node %s", storageNode.Name)
+
+			err = Inst().V.WaitDriverUpOnNode(*storageNode, addDriveUpTimeOut)
+			log.FailOnError(err, "volume drive down on node %s", storageNode.Name)
+
+			poolsAfr, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+			log.FailOnError(err, "Failed to list storage pools")
+			dash.VerifyFatal(len(poolsBfr)+1, len(poolsAfr), "verify new pool is created")
```
[the remark this answers is not in the document]
Comment (ragsingh-px) [reply tests/basic/pool_function_test.go:1592]: Done

File: tests/basic/pool_function_test.go [hunk 2]
```diff
@@ -1423,3 +1421,62 @@ var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
...
+var _ = Describe("{DriveAddDifferentTypes}", func() {
+
+	var testrailID = 34542903
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542903
+
+	BeforeEach(func() {
+		StartTorpedoTest("DriveAddDifferentTypes",
+			"Create pools with different types of drive and pool expand", nil, testrailID)
+		contexts = scheduleApps()
+	})
```
[the remark this answers is not in the document]
Comment (ragsingh-px) [reply tests/basic/pool_function_test.go:1550]: Already covered by scheduleApps() func

File: tests/basic/pool_function_test.go [hunk 3]
```diff
@@ -171,232 +173,231 @@ var _ = Describe("{PoolExpandSmoky}", func() {
...
+//				resizeErr := waitForOngoingPoolExpansionToComplete(poolIDToResize)
+//				dash.VerifyFatal(resizeErr, nil, "Pool expansion does not result in error")
+//				log.FailOnError(Inst().V.RestartDriver(*storageNode, nil),
+//					fmt.Sprintf("Error restarting px on node [%s]", storageNode.Name))
+//				log.FailOnError(Inst().V.WaitDriverUpOnNode(*storageNode, addDriveUpTimeOut),
+//					fmt.Sprintf("Timed out waiting for px to come up on node [%s]", storageNode.Name))
+//			})
+//
+//			Step("Ensure pool is up and running", func() {
+//				// Ensure pool is up and running
```
[the remark this answers is not in the document]
Comment (ragsingh-px) [reply tests/basic/pool_function_test.go]: done

**Pull request merged**